src/utils.js

```diff
@@ -86,6 +86,54 @@ var readJSON = file => {
     }
 }
 
+// FROM stackoverflow.com/questions/4253367/how-to-escape-a-json-string-containing-newline-characters-using-javascript
+var escapeJSON = str => {
+  return str
+    .replace(/[\\]/g, '\\\\')
+    .replace(/[\"]/g, '\\\"')
+    .replace(/[\/]/g, '\\/')
+    .replace(/[\b]/g, '\\b')
+    .replace(/[\f]/g, '\\f')
+    .replace(/[\n]/g, '\\n')
+    .replace(/[\r]/g, '\\r')
+    .replace(/[\t]/g, '\\t');
+};
+
+var parseHTML = file => {
```

Comment:
IMO this function should live in `export.js` because it's not a generic HTML parsing function. It's specific to parsing exported HTML.

Reply:
That is a very good point. The function was put in utils.js mostly because there was some JSON code there.